Translations/translation_FI.json

```diff
@@ -127,91 +127,91 @@
     },
     "QCMaxVoltage": {
       "displayText": "QC\njännite",
-      "description": "Ensisijainen maksimi QC jännite"
+      "description": "Ensisijainen maksimi QC jännite."
     },
     "PDNegTimeout": {
-      "displayText": "PD\ntimeout",
-      "description": "PD negotiation timeout in 100ms steps for compatibility with some QC chargers"
+      "displayText": "PD\naikakatkaisu",
+      "description": "PD neuvottelun aikakatkaisu 100ms askelin joitakin QC-latureita varten."
     },
     "USBPDMode": {
-      "displayText": "PD\nMode",
-      "description": "No Dynamic disables EPR & PPS, Safe mode does not use padding resistance"
+      "displayText": "PD\ntila",
+      "description": "Vakaa tila ei käytä EPR & PPS, turva-tila ei käytä vastuksen pehmustusta."
     },
     "BoostTemperature": {
       "displayText": "Tehostus-\nlämpötila",
-      "description": "Tehostustilan lämpötila"
+      "description": "Tehostustilan lämpötila."
     },
     "AutoStart": {
       "displayText": "Autom.\nkäynnistys",
       "description": "Käynnistää virrat kytkettäessä juotostilan automaattisesti. (J=juotostila | L=Lepotila | H=Lepotila huoneenlämpö)"
     },
     "TempChangeShortStep": {
       "displayText": "Lämmön muutos\nlyhyt painal.",
-      "description": "Lämpötilan muutos lyhyellä painalluksella"
+      "description": "Lämpötilan muutos lyhyellä painalluksella."
     },
     "TempChangeLongStep": {
       "displayText": "Lämmön muutos\npitkä painal.",
-      "description": "Lämpötilan muutos pitkällä painalluksella"
+      "description": "Lämpötilan muutos pitkällä painalluksella."
     },
     "LockingMode": {
       "displayText": "Salli nappien\nlukitus",
       "description": "Kolvatessa paina molempia näppäimiä lukitaksesi ne (V=vain tehostus | K=kaikki)"
     },
     "ProfilePhases": {
-      "displayText": "Profile\nPhases",
-      "description": "Number of phases in profile mode"
+      "displayText": "Profiili\nvaiheet",
+      "description": "Vaiheiden määrä profiilitilassa."
     },
     "ProfilePreheatTemp": {
-      "displayText": "Preheat\nTemp",
-      "description": "Preheat to this temperature at the start of profile mode"
+      "displayText": "Esilämmityksen\nlämpötila",
```

Comment:
I'm not sure, if this is going to fit, as I imagine the temp might be composed of up to 3 digits.
But since I don't own this device, I can't verify my assumption.